### PR TITLE
Migrate from npm to yarn

### PR DESCRIPTION
**Pull request by whymarrh**
This PR migrates the project from npm to yarn

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,20 +1,40 @@
...
+  test-lint:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Lint
+          command: yarn lint
```
Comment (Gudahtt) [inline .circleci/config.yml]: There is no lint script in this repo yet

Comment (whymarrh) [reply .circleci/config.yml]: 😅

Comment (whymarrh) [reply .circleci/config.yml]: Awkward

Comment (whymarrh) [reply .circleci/config.yml]: Added a placeholder

**Pull request merged**